modules/appeals_api/app/models/appeals_api/claimant.rb

```diff
@@ -0,0 +1,117 @@
+class Claimant
+  def initialize(auth_headers, form_data)
+    @auth_headers = auth_headers
+    @form_data = form_data
+  end
+
+  def first_name
+    first_name_header
```

Comment:
Not really sure why this method layering needs to happen. Why not use `auth_headers['X-Claimant-First-Name']` here?

Reply:
I was trying something out it's being reverted today